WPFSKillTree/Utils/Extensions/NumberExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace POESKillTree.Utils.Extensions
+{
+    public static class NumberExtensions
+    {
+        public static bool AlmostEquals(this float x, float y)
+        {
+            // float has 6 significant digits. 4 leaves a bit of room for accumulated errors.
+            var epsilon = Math.Max(Math.Abs(x), Math.Abs(y)) * 1e-5F;
+            return AlmostEquals(x, y, epsilon);
+        }
+        public static bool AlmostEquals(this float x, float y, float epsilon)
+        {
+            return Math.Abs(x - y) <= epsilon;
+        }
+
+        public static bool AlmostEquals(this double x, double y)
+        {
+            // double has 15 significant digits. 13 leaves a bit of room for accumulated errors.
+            var epsilon = Math.Max(Math.Abs(x), Math.Abs(y)) * 1e-14;
+            return AlmostEquals(x, y, epsilon);
```

Comment:
Thanks, didn't know that.

These were used exactly once, so I'll keep it simple and just remove the relative error ones. In most cases, it should be easy to manually set a delta that makes sense in the context (e.g. stats only use up to 2-3 decimal places).